tests/integration/test_logcollector/test_configuration/test_basic_configuration_ignore_binaries.py

```diff
@@ -64,7 +64,9 @@
 from wazuh_testing.tools.services import control_service
 from wazuh_testing.tools.file import truncate_file
 import wazuh_testing.api as api
-from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX, WINDOWS_AGENT_DETECTOR_PREFIX, FileMonitor
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.modules.logcollector import LOG_COLLECTOR_PREFIX, WINDOWS_AGENT_PREFIX
```

Comment:
These constants renaming were not populated in the code. See:
https://github.com/wazuh/wazuh-qa/pull/4263/files#diff-bb0af03f6d265932e007df345ab06f3bfa0c756c679b388667e4f7cf4c507868R88
https://github.com/wazuh/wazuh-qa/pull/4263/files#diff-bb0af03f6d265932e007df345ab06f3bfa0c756c679b388667e4f7cf4c507868R91

Reply:
- Applied fix in https://github.com/wazuh/wazuh-qa/pull/4263/commits/7249189978560208455c8e11bbf18a6023187be2

## Fix Evidence

| Local | Jenkins |
|:--|:--|
| [:green_circle:](https://github.com/wazuh/wazuh-qa/files/12177247/Logcollector.Local.zip) | [:green_circle:](https://github.com/wazuh/wazuh-qa/files/12177249/manager_html_report_Test_integration_B41704_20230726211927.zip)